### PR TITLE
fixed cyclic lr state dict

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/training/utils/checkpoint_utils.py [hunk 1]
```diff
@@ -1597,3 +1598,16 @@ def load_pretrained_weights_local(model: torch.nn.Module, architecture: str, pre
...
+
+def get_scheduler_state(scheduler) -> Dict:
+    """
+    Wrapper for getting a torch lr scheduler state dict, resolving some issues with CyclicLR
+    (see https://github.com/pytorch/pytorch/pull/91400)
+    :param scheduler: torch.optim.lr_scheduler._LRScheduler, the scheduler
+    :return: the scheduler's state_dict
+    """
+    state = scheduler.state_dict()
+    if isinstance(scheduler, CyclicLR) and int(torch.version.__version__.split(".")[0]) < 2:
```
Comment (BloodAxe) [inline src/super_gradients/training/utils/checkpoint_utils.py:1611]: `torch_version_is_greater_or_equal` 🙏 🙏 🙏 
We have a fancy method for that )

Comment (shaydeci) [reply src/super_gradients/training/utils/checkpoint_utils.py]: added local import because of discussed circular import of it in latest commit.

**Review by BloodAxe — APPROVED** [review]
LGTM

**Comment by BloodAxe** [comment]
Closing this one in favor of https://github.com/Deci-AI/super-gradients/pull/1518 which contains one cherry-picked commit
